src/main/java/uk/gov/pay/directdebit/notifications/clients/AdminUsersClient.java

```diff
@@ -27,20 +30,31 @@ public AdminUsersClient(AdminUsersConfig config, Client client) {
 
     public void sendEmail(EmailTemplate template, Mandate mandate, Map<String, String> personalisation) {
         String gatewayAccountExternalId = mandate.getGatewayAccount().getExternalId();
-        LOGGER.info("Calling adminusers to send {} email for mandate id {} for gateway account id {}",
-                template.toString(),
-                mandate.getExternalId(),
-                gatewayAccountExternalId);
         String email = mandate.getPayer().orElseThrow(() -> new PayerNotFoundException(mandate.getExternalId())).getEmail();
         var emailPayloadRequest = new EmailPayloadRequest(email, gatewayAccountExternalId, template, personalisation);
         try {
-             Response response = client.target(config.getAdminUsersUrl())
+            Response response = client.target(config.getAdminUsersUrl())
                     .path("/v1/emails/send")
                     .request()
                     .post(Entity.entity(emailPayloadRequest, MediaType.APPLICATION_JSON_TYPE));
-            response.readEntity(String.class);
+            if (response.getStatus() != 200) {
+                LOGGER.info("Call to adminusers to send {} email for mandate id {} for gateway account id {} failed with status {}, body {}",
```

Comment:
should this level be `Warning` rather than `info`? Seems this is something we might want to differentiate from the norm.

Reply:
I think so. I blindly copypasted it

Reply:
Thinking about it again, should this also be `Error` similar to the exception case. Shouldn't we always investigate why an email that DDConnector needs to send to a payee isn't sent, assuming that anything but 200 means they won't be getting that email which is probably important to them. Isn't there some regulatory requirement to send payment details (seem to remember something about having to provide SUN).